internal/pipe/git/git.go

```diff
@@ -65,6 +66,10 @@ func getInfo(ctx *context.Context) (context.GitInfo, error) {
 }
 
 func getGitInfo() (context.GitInfo, error) {
+	branch, err := getBranch()
+	if err != nil {
+		return context.GitInfo{}, errors.Wrap(err, "couldn't get current branch")
```

Comment:
```suggestion
		return context.GitInfo{}, fmt.Errorf("couldn't get current branch: %w", err)
```

Reply:
Thanks, I started building this on a branch before this new pattern was added.